index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="styles/header.css">
+  <link rel="stylesheet" href="styles/headline.css">
+  <link rel="stylesheet" href="styles/general.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="header">
+    <nav class="nav">
+      <div class="left-section">
+        <p class="logo">My Logo</p>
+      </div>
+      <button class="right-section">
+        <img class="icon-menu" src="/Normal Button/Tertiary/Icons/Icon - Menu.png" alt="">
+      </button>
+    </nav>
+    <section class="headline">
+      <h2 class="introduction">I’m Mariane
+        Glad to see you!</h2>
+      <p class="about">I’m a software developer! I can help you build a product , 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.
+      </p>
+      <p class="supporting-text">LET'S CONNECT</p>
+      <div class="social-media-icons">
+        <a href="#">
+          <img src="images/Vector.svg" alt="">
+        </a>
+        <a href="#">
+          <img src="images/Linkedin icon.svg" alt="">
+        </a>
+        <a href="#">
+          <img src="images/Vector (1).svg" alt="">
+        </a>
+        <a href="#">
+          <img src="images/Vector (2).svg" alt="">
+        </a>
+        <a href="#">
+          <img src="images/Vector (3).svg" alt="">
+        </a>
+      </div>
```

Comment:
- [ ] Great job using `<a> and <img>`  tags However I think it would be also nice to make it as a list  icon 
The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.
 
check this example below :+1: 
 ```
<ul>
                        <li><a href="#"><i class="fa fa-facebook fa-2x"></i></a></li>
                        <li><a href="#"><i class="fa fa-twitter fa-2x"></i></a></li>
                        <li><a href="#"><i class="fa fa-github fa-2x"></i></a></li>
                        <li><a href="#"><i class="fa fa-instagram fa-2x"></i></a></li>
                        <li><a href="#"><i class="fa fa-linkedin fa-2x"></i></a></li>
</ul>
```

Reply:
I have wrapped all the social media links in to un-ordered list.

Thank you for the examples.